### DISCUSSION TITLE
Is Flood Fill Algorithm included in ImageSharp?

### DISCUSSION THREAD

**Discussion by atuladhar** [SixLabors/ImageSharp#1486] [Q&A] [answered]
Does ImageSharp have built-in Flood Fill Algorithms or any related functions included?

**Comment by antonfirsov** [comment, marked as answer]
@atuladhar no unfortunately, but I think its rrlatively easy to implement a basic variant in your app.

It might be a feature in-scope for ImageSharp.Drawing though, personally I would be fine with a community contribution adding it there.
